packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -372,6 +372,48 @@ export class MockContainerRuntime extends TypedEventEmitter<IContainerRuntimeEve
 		});
 	}
 
+	/**
+	 * If flush mode is set to FlushMode.TurnBased, it will send the specified number of messages from the outbox
+	 * queued since the last time this (or flush) was called. Otherwise, calling the method does nothing.
+	 * This can be useful when simulating staging mode, and we only want to flush certain messages.
+	 */
+	public flushSomeMessages(numMessages: number): void {
```

Comment:
we should update flush to call this with the totally count, rather than duplicate

Reply:
Good call - updated in latest